twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/QueryCompletedEventScriber.java

```diff
@@ -93,6 +116,13 @@ private static QueryCompletionEvent toThriftQueryCompletionEvent(QueryCompletedE
       thriftEvent.distributed_planning_time_ms = eventStat.getDistributedPlanningTime().get().toMillis();
     }
     thriftEvent.total_bytes = eventStat.getTotalBytes();
+    if (eventMetadata.getPayload().isPresent()) {
+      JsonReader jsonReader = Json.createReader(new StringReader(eventMetadata.getPayload().get()));
+      long totalDataTransferBytes = getTotalDataTransferInBytes(jsonReader.readObject());
+      // uncomment after new thrift def is built and uploaded
+      //thriftEvent.total_data_transfer_bytes = totalDataTransferBytes;
```

Comment:
For semantic clarity, if we're summing up output data, why not call it `totalDataTransferOutputBytes`? This makes it very clear that this doesn't include the input of the first stage, and that it doesn't double count input/output of each stage. Without that naming, I'd wonder about those things when seeing `totalDataTransferBytes` in our metrics table without context.

Also keep in mind that at some point we will support presto writing back to HDFS, in which case we might want to track output of those stages separately. Can you plan for that in the current approach?